core/templates/dev/head/pages/exploration_editor/editor_tab/StateHints.js

```diff
@@ -51,6 +75,40 @@ oppia.controller('StateHints', [
       return hintAsPlainText;
     };
 
+    $scope.getQuittersPercent = function(currentHintIndex) {
```

Comment:
Pull this and the next method into a HintStats domain object, and write robust unit tests for them.